packages/mask/src/plugins/ITO/SNSAdaptor/SwapDialog.tsx

```diff
@@ -172,12 +172,20 @@ export function SwapDialog(props: SwapDialogProps) {
             const { to_value } = (receipt.events?.SwapSuccess?.returnValues ?? {}) as {
```

Comment:
The type and value are mismatched.

```suggestion
            const { to_value } = (receipt.events?.SwapSuccess?.returnValues ?? { to_value: 0}) as {
```